src/components/HelpScoutBeacon/HelpScoutBeacon.js

```diff
@@ -64,12 +64,11 @@ const Beacon = React.memo(function Beacon({ locator, apps }) {
         position: absolute;
         bottom: ${2 * GU}px;
         right: ${2 * GU}px;
-        z-index: 4;
+        z-index: 2;
 
         ${breakpoint(
-          'medium',
+          'large',
           `
-            z-index: 2;
```

Comment:
Just wanted to double check that this is safe, since we've had a couple of issues with the Help Scout button's z-index.

Reply:
This was being used inside the media query for larger screens, mobile had a different value because of the menu panel, but this time around the menu panel will close before the Help Scout popover opens, so it is safe to have the same `z-index` for all screens.